types/leaflet/index.d.ts

```diff
@@ -2923,7 +2923,7 @@ export class Map extends Evented {
     getPane(pane: string | HTMLElement): HTMLElement | undefined;
     getPanes(): { [name: string]: HTMLElement } & DefaultMapPanes;
     getContainer(): HTMLElement;
-    whenReady(fn: () => void, context?: any): this;
+    whenReady(fn: (event: LeafletEvent) => void, context?: any): this;
```

Comment:
Can you explain this change? It doesn't seem to correspond to [the source](https://github.com/Leaflet/Leaflet/blob/f4eeb7feea2038058cf7d61b4e1ba2c47a2b3297/src/map/Map.js#L1479)

Reply:
@someonewithpc I was thinking to offer an event related type, but, by the typescript book, `target` property is already correct and is not adding extra noise.